examples/hpc/dla-driver/src/layers.rs

```diff
@@ -0,0 +1,277 @@
+#![no_std]
+
+use crate::tensor3::{Order3, Tensor3};
+use crate::tensor4::{Order4, Tensor4};
+use crate::{
+    Dla, InputSize, KernelSize, LayerConfig, MemoryBank, Padding, SimdBitMode, Stride,
+    MEMORY_BANK_10_OFFSET, MEMORY_BANK_BASE_ADDR,
+};
+use alloc::vec::Vec;
+
+use crate::utils::{
+    calculate_conv2d_out_param_dim, calculate_number_of_banks_needed, get_banks_for_layer,
+};
+
+#[derive(Clone, Copy)]
+enum DlaOutput {
+    I8,
+    I16,
+    I32,
+}
```

Comment:
It looks like this is not used, at least in this file. I would remove this, and rename `DlaOutputType` as `DlaOutput`.